### ISSUE TITLE
Change config.php delivery

### ISSUE THREAD

**Issue by wjbeckett** [Monitorr/Monitorr#46]
Need to change the way we supply the config file so that updates do not overwrite a users config.

- [x] Change the config.php to config.php.sample
- [x] Update README to reflect the new config setup (copy config.in.sample to config.ini)

**Comment by wjbeckett** [comment]
Completed in  https://github.com/Monitorr/Monitorr/commit/143abe204b34e3d5e295b279c90ff58944f9b9d5

**Comment by seanvree** [comment]
@wjbeckett  I was thinking to add a mirrored copy of conifg.php.samlpe in the form of config.php.  That way it runs out of the box.  When we make changes to the config, we make changes in BOTH files....so if current users pull, they will get an error saying they need to update their config.php file. If it's a NEW users, they'll be good to go. 

I just like the fact that if we have a config.php that it will run straight out of the box when cloning. 

thoughts?

**Comment by seanvree** [comment]
Yup, did some testing, we should DEF add a config.php that mirrors the config.php.sample. 

without a config.php in the repo  - If user copied the config.php.sample THEN renamed it to config.php (which is what they should do), when they PULL an update, they will NOT know there's been an update to the config file unless they manually compare the two.    However, if there is an config.php file, when they go to pull,  they will get an error notifing them the config.php file has been updated, but it won't overwrtie it. However, the NEW changes will be pushed to the config.php file, so they can compare on their own and update.  

Make sense?

thoughts?

**Comment by wjbeckett** [comment]
Let's do it.

**Comment by seanvree** [comment]
merged to develop. 

https://github.com/Monitorr/Monitorr/commit/f867c50793191efbc4204cbcba2a8ea8b24fcfeb

Closed

**Comment by jonfinley** [comment]
Reconfiguring for #24

**Comment by jonfinley** [comment]
allows for name, link, icon selection
```
$myServices = array( 
    "monitorr1" => array( 
        "link" => "https://finflix.net/monitorr", 
        "image" => "monitorr.png"
        ), 
    "plex2" => array( 
        "link" => "https://plex.finflix.net", 
        "image" => "plex.png"
        ), 
    "sonarr" => array(  
        "link" => "https://finflix.net/sonarr", 
        "image" => "sonarr.png"
        ), 
    "radarr4" => array( 
        "link" => "https://finflix.net/radarr", 
        "image" => "radarr.png"
        ), 
   ); 
```

**Comment by jonfinley** [comment]
Merged #56
